tripper/triplestore.py

```diff
@@ -342,28 +344,28 @@ def serialize(
                 serialisation is returned.
             format: Format to serialise as.  Supported formats, depends on
                 the backend.
-            backend: If given, use the parse() method of the specified
-                backend instead of the current backend.
-            backend_kwargs: Dict with additional keyword arguments passed
-                when instantiating `backend`.
+            fallback_backend: If the current backend doesn't implement
+                serialisation, use the `fallback_backend` instead.
+            fallback_backend_kwargs: Dict with additional keyword arguments
+                for initialising `fallback_backend`.
```

Comment:
Same comment as for parse above

Reply:
Tested in tests/backends/test_collection.py